app/classifier/classifier.spec.js

```diff
@@ -20,8 +20,14 @@ const store = {
   })
 };
 
+const geordi = {
+  keys: {},
+  forget: sinon.stub(),
+  remember: sinon.stub()
+};
+
 const mockReduxStore = {
-  context: { store },
+  context: { geordi, store },
```

Comment:
Why is geordi being added here?

Reply:
`handleSubjectImageLoad` crashes if `geordi` is undefined so I mocked it in the tests.